router/utils/utils.go

```diff
@@ -17,6 +17,10 @@ var (
 	JobRetention time.Duration
 )
 
+const (
+	DRAIN_ERROR_CODE int = 401
```

Comment:
any specific reason why we chose HTTP 401 - Unauthorized for drained jobs?

Reply:
No, I think I made a mistake, instead of using 410 that you suggested earlier, I ended up using 401. Let me change it to 410.